x/staking/handler_test.go

```diff
@@ -500,13 +496,11 @@ func TestIncrementsMsgUnbond(t *testing.T) {
 		require.NoError(t, err)
 		require.NotNil(t, res)
 
-		ts := &gogotypes.Timestamp{}
-		types.ModuleCdc.MustUnmarshalBinaryLengthPrefixed(res.Data, ts)
-
-		finishTime, err := gogotypes.TimestampFromProto(ts)
+		var resData types.MsgUndelegateResponse
+		err = proto.Unmarshal(res.Data, &resData)
```

Comment:
maybe `app.AppCodec().UnmarshalBinaryBare` is even more correct

Reply:
Well that won't work with amino though. I think it's okay